src/Resources/skeleton/validator/Validator.tpl.php

```diff
@@ -7,7 +7,7 @@
 
 class <?= $class_name ?> extends ConstraintValidator
 {
-    public function validate($value, Constraint $constraint)
+    public function validate(mixed $value, Constraint $constraint): void
```

Comment:
I'm not sure if this is possible (e.g. maker has enough knowledge about `$value`) but can we do `string|array<int, mixed>|int|etc.....` here instead of `mixed`? Thinking about phpstan level 9?  where it complains if something is `mixed` or if an `array` does not define its shape.

I'm not suggesting we do all potential types as a union, but if we know `$value` could be say, either a `string` or a `int` - we should use `string|int` as the type instead of mixed.

else `mixed` is perfectly fine here :wink: